pkg/server/server.go

```diff
@@ -351,6 +386,15 @@ func MustNewServerWithOpts(opts ...OpenFGAServiceV1Option) *Server {
 	return s
 }
 
+func (s *Server) IsExperimentallyEnabled(flag ExperimentalFeatureFlag) bool {
+	for _, e := range s.experimentals {
```

Comment:
```suggestion
	return slices.Contains(s.experimentals, flag)
```

This whole block of code can reduce to using `slices.Contains`.